docs/openj9_support.md

```diff
@@ -147,7 +147,7 @@ OpenJDK 11 binaries are supported on the minimum operating system levels shown i
 When public support for an operating system version ends, OpenJ9 can no longer be supported on that level.
 
 
-### OpenJDK 14
+### OpenJDK 15
```

Comment:
There is a "OpenJDK 14" a few lines down.

Reply:
Fixed